controller/internal/routes/identity_router.go

```diff
@@ -46,187 +49,248 @@ func NewIdentityRouter() *IdentityRouter {
+	Patch(rc, func(id string, fields JsonFields) error {
+		return ae.Handlers.Identity.Patch(MapPatchIdentityToModel(params.ID, params.Body, getIdentityTypeId(ae, params.Body.Type)), fields.FilterMaps("tags"))
 	})
 }
 
-func (ir *IdentityRouter) listEdgeRouterPolicies(ae *env.AppEnv, rc *response.RequestContext) {
-	ListAssociationWithHandler(ae, rc, ir.IdType, ae.Handlers.Identity, ae.Handlers.EdgeRouterPolicy, MapEdgeRouterPolicyToApiEntity)
+func (r *IdentityRouter) listEdgeRouterPolicies(ae *env.AppEnv, rc *response.RequestContext) {
+	ListAssociationWithHandler(ae, rc, ae.Handlers.Identity, ae.Handlers.EdgeRouterPolicy, MapEdgeRouterPolicyToRestEntity)
 }
 
-func (ir *IdentityRouter) listServicePolicies(ae *env.AppEnv, rc *response.RequestContext) {
-	ListAssociationWithHandler(ae, rc, ir.IdType, ae.Handlers.Identity, ae.Handlers.ServicePolicy, MapServicePolicyToApiEntity)
+func (r *IdentityRouter) listServicePolicies(ae *env.AppEnv, rc *response.RequestContext) {
+	ListAssociationWithHandler(ae, rc, ae.Handlers.Identity, ae.Handlers.ServicePolicy, MapServicePolicyToRestEntity)
 }
 
-func (ir *IdentityRouter) listServices(ae *env.AppEnv, rc *response.RequestContext) {
+func (r *IdentityRouter) listServices(ae *env.AppEnv, rc *response.RequestContext) {
 	filterTemplate := `not isEmpty(from servicePolicies where anyOf(identities) = "%v")`
-	ListAssociationsWithFilter(ae, rc, ir.IdType, filterTemplate, ae.Handlers.EdgeService, MapServiceToApiEntity)
+	ListAssociationsWithFilter(ae, rc, filterTemplate, ae.Handlers.EdgeService, MapServiceToRestEntity)
 }
 
-func (ir *IdentityRouter) listEdgeRouters(ae *env.AppEnv, rc *response.RequestContext) {
+func (r *IdentityRouter) listEdgeRouters(ae *env.AppEnv, rc *response.RequestContext) {
 	filterTemplate := `not isEmpty(from edgeRouterPolicies where anyOf(identities) = "%v")`
-	ListAssociationsWithFilter(ae, rc, ir.IdType, filterTemplate, ae.Handlers.EdgeRouter, MapEdgeRouterToApiEntity)
+	ListAssociationsWithFilter(ae, rc, filterTemplate, ae.Handlers.EdgeRouter, MapEdgeRouterToRestEntity)
 }
 
-func (ir *IdentityRouter) listServiceConfigs(ae *env.AppEnv, rc *response.RequestContext) {
-	listWithId(rc, ir.IdType, func(id string) ([]interface{}, error) {
-		configs, err := ae.Handlers.Identity.GetServiceConfigs(id)
+func (r *IdentityRouter) listServiceConfigs(ae *env.AppEnv, rc *response.RequestContext) {
+	listWithId(rc, func(id string) ([]interface{}, error) {
+		serviceConfigs, err := ae.Handlers.Identity.GetServiceConfigs(id)
 		if err != nil {
 			return nil, err
 		}
 		result := make([]interface{}, 0)
-		for _, config := range configs {
-			result = append(result, IdentityServiceConfig{Service: config.Service, Config: config.Config})
+		for _, serviceConfig := range serviceConfigs {
+			service, err := ae.Handlers.EdgeService.Read(serviceConfig.Service)
+			if err != nil {
+				continue
```

Comment:
not even a debug log would be useful? if not - that's cool just askin'

Reply:
A debug would be nice.